sormas-backend/src/test/java/de/symeda/sormas/backend/event/EventParticipantFacadeEjbTest.java

```diff
@@ -551,6 +552,38 @@ public void searchEventParticipantsByPersonOtherDetail() {
 		eventParticipantIndexDtos = getEventParticipantFacade().getIndexList(eventParticipantCriteria, 0, 100, null);
 		assertEquals(0, eventParticipantIndexDtos.size());
 	}
+	@Test
+	public void testCreationOfEventParticipantWhenArchived()
+	{
```

Comment:
Code formatting violated: Use Code formatting to move `{` after the method declaration